src/code/operation.js

```diff
@@ -656,32 +650,32 @@ export default class WasabeeOp {
       return null;
     }
 
+    if (!options) options = {};
```

Comment:
wouldn't this be better in the method def?

addLink(fromPortal, toPortal, options = {})